Geometry/MTDGeometryBuilder/interface/RectangularMTDTopology.h

```diff
@@ -175,6 +159,14 @@ class RectangularMTDTopology final : public PixelTopology {
   int colsperroc() const override { return m_COLS_PER_ROC; }
   float xoffset() const { return m_xoffset; }
   float yoffset() const { return m_yoffset; }
+  float gapxInterpad() const { return m_GAPxInterpad; }
+  float gapyInterpad() const { return m_GAPyInterpad; }
+  float gapxBorder() const { return m_GAPxBorder; }
+  float gapyBorder() const { return m_GAPyBorder; }
```

Comment:
Please add a comment here that the border and interpad values are returned as cm by these methods.